_po/ja/how-to/laravel.po

```diff
@@ -403,6 +584,16 @@ msgid ""
 "|\n"
 "*/"
 msgstr ""
+"/*\n"
+"|--------------------------------------------------------------------------\n"
+"| Web Routes\n"
+"|--------------------------------------------------------------------------\n"
+"|\n"
+"| Here is where you can register web routes for your application. These\n"
+"| routes are loaded by the RouteServiceProvider and all of them will\n"
+"| be assigned to the \"web\" middleware group. Make something great!\n"
+"|\n"
+"*/"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.